kernel/etc/Shell.c

```
@@ -43,7 +43,7 @@
 
 static char command_buffer[256];
 static int cmd_pos = 0;
-static char current_dir[256] = "/";
+char current_dir[256] = "/";
```

Clang (14.0.6)

[warning] variable 'current_dir' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)